evergreen.yml

```diff
@@ -341,6 +342,21 @@ functions:
           CMAKE: "${cmake|cmake}"
           PATH: "${toolchain}:/usr/bin:/bin"
 
+  ##
+  # Runs each workload as a dry run
+  # Requires f_compile to have been run first.
+  ##
+  f_dry_run_workloads:
+  - command: subprocess.exec
+    params:
+      working_dir: .
+      binary: /bin/bash
+      args:
+        - "${workdir}/src/scripts/dry-run-workloads.sh"
+        - "${workdir}/dist/etc/genny/workloads"
```

Comment:
why require an `install` first and not just do this for `src/driver/test`? (We'll change that in a separate ticket but it is what it is for now)

Reply:
I'm mildly eyeing the cases where a) we have certain workloads that we for some reason don't want to test and b) we're separating our workloads somehow from our testing code. Essentially, I view dist as a mutable artifact and source as a fact of the current evergreen task. It's not very important right now.

Reply:
Okay seems reasonable